### PR TITLE
Install go tools in root Makefile from golang.org

### PR DESCRIPTION
**Pull request by mboersma**
[code.google.com is going away](http://google-opensource.blogspot.com/2015/03/farewell-to-google-code.html), and there were multiple versions of `make setup-gotools` in the project. This also moves the location of the `websocket` package used by logspout.

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -345,6 +340,10 @@
 			"Rev": "f7445b17d61953e333441674c2d11e91ae4559d3"
 		},
 		{
+			"ImportPath": "golang.org/x/net/websocket",
+			"Rev": "d9558e5c97f85372afee28cf2b6059d7d3818919"
```
Comment (bacongobbler) [inline Godeps/Godeps.json]: How come we're bumping the rev here to master? https://github.com/golang/net/commits/4231557d7c726df4cf9a4e8cdd8a417c8c200bdb doesn't seem to exist so I'm not sure what we were currently pointing at, but if you're confident that this won't break stuff then I'm ok with this

**Comment by bacongobbler** [comment]
code LGTM.

File: Godeps/Godeps.json (hunk 1 above)
Comment (mboersma) [inline Godeps/Godeps.json]: I couldn't match up the SHAs between hosts, so I just vendored from master and looked at the diff. I agree, it looks like safe changes that logspout probably wouldn't care about anyway. :four_leaf_clover:

**Comment by carmstrong** [comment]
Code LGTM.

**Pull request merged**